scripts/test_env.sh

```diff
@@ -7,62 +7,66 @@ TEST=${TEST:-"./test"}
 HOST=${HOST:-"127.0.0.1"}
```

Comment:
This code is no longer what I'd call "reasonably small to be best done in `bash`". 

We should consider looking at other alternatives, e.g. `python`: 
https://unix.stackexchange.com/questions/24802/on-which-unix-distributions-is-python-installed-as-part-of-the-default-install

@astrokov7 had the idea of rewriting tests using `python`'s subprocess infrastructure, it might be worth it to do the same here.